rl-sideswipe-access/src/components/PermissionOverlay_old.tsx

```
@@ -0,0 +1,242 @@
+
+  const checkPermissions = async () => {
+    try {
+      const status = await NativeControl.getDetailedPermissionStatus();
+      setPermissionStatus(status);
+      console.log('Detailed permission status:', status);
+    } catch (error) {
+      console.error('Failed to check permissions:', error);
+    }
+  };
+
+  const handleEnableAccessibility = handleAccessibilitySettings;
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)